cpp/tests/experimental/katz_centrality_test.cpp

```diff
@@ -39,7 +40,7 @@
 // do the perf measurements
 // enabled by command line parameter s'--perf'
 //
-static int PERF = 0;
+static int PERF = 1;
```

Comment:
should this be 0 or 1 by default? I think we have the command line parameter `--perf` to control this.

Reply:
Yes, correct, this should be 0 by default, I will make a fix.